service-app/internal/parser/interface.go

```diff
@@ -1,11 +1,12 @@
 package parser
 
 type CommonValidator interface {
-	Setup(doc interface{}) error
+	Setup(doc any) error
 	Validate() error
+	GetValidatorErrorMessages() []string
```

Comment:
It does seem weird having the exact same definition in every validator file, but I don't have an idea for a solution for that 😄

Reply:
Your right it seems redundant, it was the simplest approach at the time but I feel like we can embed the base struct somehow. I'll investigate

Reply:
https://github.com/ministryofjustice/opg-scanning/pull/81/commits/de243125876f54a9df076d72caf2fa2391d57e46